yew-functional-macro/tests/functional_macro/bad-return-type-fail.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0308]: mismatched types
+  --> $DIR/bad-return-type-fail.rs:10:24
+   |
+10 |   fn comp(props: &Props) {
```

Comment:
It's the same error, just with different type.  
```rust
#[functional_component(Comp)]
fn comp(props: &Props) -> u32 {
    let _x = html! {
        <p>
            { props.a }
        </p>
    };
    1
}
```
`stderr` file:
```
error[E0308]: mismatched types
  --> $DIR/bad-return-type-fail.rs:16:5
   |
9  | #[functional_component(Comp)]
   | ----------------------------- expected `yew::virtual_dom::vnode::VNode` because of return type
...
16 |     1
   |     ^ expected enum `yew::virtual_dom::vnode::VNode`, found integer
```